charts/azure-k8s-metrics-adapter/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: azure-k8s-metrics-adapter
+version: 0.1.0
```

Comment:
What are the differences between the version numbers?

Reply:
`version` is a SemVer2 version that identifies the version of the Chart whereas `appVersion` is an informational field about the version included with the chart at the specified `version`